examples/notebooks/acrobot_urdf.ipynb

```diff
@@ -0,0 +1,453 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# The Acrobot\n",
+    "\n",
+    "*We recommend you look at the [Introduction to Crocoddyl](introduction_to_crocoddyl.ipynb) example before this one.*\n",
+    "\n",
+    "In the example, we model the acrobot control problem using Crocoddyl. An acrobot is a two joint planar robot with only one actuator. It is a canonnical example of an underactuated system and so presents an interesting control problem.\n",
+    "\n",
+    "We demonstrate how to:\n",
+    "\n",
+    "1. Load a model from an urdf.\n",
+    "1. Define an actuation mapping for the system.\n",
+    "1. Construct and solve the control problem.\n",
+    "\n",
+    "## Loading the model\n",
+    "A standalone double pendulum robot urdf is provided in the [example-robot-data](https://github.com/Gepetto/example-robot-data) repository, this comes bundled with Crocoddyl. Let's load the model and inspect its properties."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Nb joints = 3 (nq=2,nv=2)\n",
+      "  Joint 0 universe: parent=0\n",
+      "  Joint 1 joint1: parent=0\n",
+      "  Joint 2 joint2: parent=1\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "import numpy as np\n",
+    "import pathlib\n",
+    "import crocoddyl\n",
+    "import pinocchio\n",
+    "\n",
+    "# Get the path to the urdf\n",
+    "from example_robot_data.path import EXAMPLE_ROBOT_DATA_MODEL_DIR\n",
+    "urdf_model_path = pathlib.Path('double_pendulum_description', 'urdf', 'double_pendulum_simple.urdf')\n",
+    "urdf_model_path = os.path.join(EXAMPLE_ROBOT_DATA_MODEL_DIR, urdf_model_path)\n",
+    "\n",
+    "# Now load the model (using pinocchio)\n",
+    "robot = pinocchio.robot_wrapper.RobotWrapper.BuildFromURDF(str(urdf_model_path))\n",
```

Comment:
Better to use the `load` function.
See the feedback in `example-robot-data`

Reply:
It was difficult to find the correct way to load an urdf that was not included in `example-robot-data`. Especially as I am not familiar with `pinocchio`. It might be useful to have an example that demonstrates the common use case of loading an urdf and having the ability to view it. But maybe that should be a separate example?

Sources:
[overview-rudf.py](https://github.com/stack-of-tasks/pinocchio/blob/master/examples/overview-urdf.py) uses `buildModelFromUrdf`.
[robot-wrapper-viewer.py](https://github.com/stack-of-tasks/pinocchio/blob/master/examples/robot-wrapper-viewer.py) uses `RobotWrapper.BuildFromURDF` which wraps `buildModelsFromUrdf`. (note "Models" not "Model").
There is also `pinocchio.buildModelsFromUrdf` (XML, String) which uses `buildModelFromUrdf` and `buildGeomFromUrdf`.
And [robots_loader.py](https://github.com/Gepetto/example-robot-data/blob/master/python/example_robot_data/robots_loader.py) calls to `RobotWrapper.BuildFromURDF`. 
Related to https://github.com/Gepetto/example-robot-data/issues/5.

I advocate for keeping it as is but happy to use `load` if you think it should be consistent?